src/main/java/org/folio/linked/data/model/entity/InstanceMetadata.java

```diff
@@ -0,0 +1,54 @@
+package org.folio.linked.data.model.entity;
+
+import static lombok.AccessLevel.PROTECTED;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.Accessors;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+
+@Entity
+@Data
+@NoArgsConstructor(access = PROTECTED)
+@Accessors(chain = true)
+@Table(name = "instance_metadata")
+@EqualsAndHashCode(of = "id")
+public class InstanceMetadata {
+
+  @Id
+  @Column(name = "resource_hash")
+  private Long id;
+
+  @Column(name = "inventory_id")
+  private String inventoryId;
+
+  @Column(name = "srs_id")
+  private String srsId;
+
+  @Column(name = "source")
+  @Enumerated(EnumType.STRING)
+  @JdbcTypeCode(SqlTypes.NAMED_ENUM)
```

Comment:
What's the purpose of the annotation? Usually it's enough to put the @Enumerated(EnumType.STRING)

Reply:
It is needed since we have a custom type defined in the database  (see: src/main/resources/changelog/scripts/v-1.0.0/resource_graph/types/resource_source.sql). Without this annotation all tests fails with error 

ERROR: column "source" is of type resource_source but expression is of type smallint

Reply:
I see, thanks for explanation!

Reply:
Actually, @Enumerated was not needed any more. Removed it.